nat-tester/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if [ "$(basename "$PWD")" = "scion" ]; then
+    cd nat-tester || return
+fi
+
+if [ "$(basename "$PWD")" != "nat-tester" ]; then
+    echo "This script must be run from within the scion directory"
+    exit 1
+fi
+
+CGO_ENABLED=0 go build -o "test-server" ./test-server.go
+CGO_ENABLED=0 go build -o "snet-test-client" ./snet-test-client.go
```

Comment:
I like this approach so much more than using Bazel. On the other hand, aren't all the other tests also supported via Bazel? We should probably do that as well.

Reply:
Indeed, Bazel is also used on the CI pipeline, we need to integrate them in Bazel.